docs/FAQ

```diff
@@ -588,7 +588,13 @@ FAQ
   Exactly what kind of quotes and how to do this is entirely up to the shell
   or command line interpreter that you are using. For most unix shells, you
   can more or less pick either single (') or double (") quotes. For
-  Windows/DOS prompts I believe you are forced to use double (") quotes.
+  Windows/DOS command prompts you must use double (") quotes, and if you have
+  inner double quotes you can escape them with a backslash.
+
+  For Windows powershell the arguments are not always passed on as expected
+  because curl is not a powershell script. You may or may not be able to use
+  single quotes. To escape inner double quotes seems to require a
+  backslash-backtick escape sequence and the outer quotes as double quotes.
```

Comment:
Does this warrant a small example, or is that overkill?

Reply:
I think it's overkill. It's already a stretch to explain that inner double quotes need escaping.